code/__DEFINES/_profile.dm

```diff
@@ -1,4 +0,0 @@
-#if DM_BUILD >= 1506
-// We don't actually care about storing the output here, this is just an easy way to ensure the profile runs first.
-GLOBAL_REAL_VAR(world_init_profiler) = world.Profile(PROFILE_RESTART)
```

Comment:
@LemonInTheDark is working on updating this file, why was this removed?

Reply:
Moved to world where init order is defined.

It doesn't even care about the retval. It's just doing it for init order.

This PR supercedes that change